assets/js/pagination-util.js

```diff
@@ -54,17 +75,23 @@ function changePageUtil(curr, index) {
 }
 
 function displayNavArrows(i) {
+  var farLeft = document.querySelector(".pagination .bx.bx-chevrons-left");
   var left = document.querySelector(".pagination .sgds-icon.sgds-icon-arrow-left");
+  var farRight = document.querySelector(".pagination .bx.bx-chevrons-right");
   var right = document.querySelector(".pagination .sgds-icon.sgds-icon-arrow-right");
 
   if (i === 0) {
+    if (farLeft) farLeft.classList.add("sgds-icon-disabled");
     left.classList.add("sgds-icon-disabled");
   } else {
+    if (farLeft) farLeft.classList.remove("sgds-icon-disabled");
     left.classList.remove("sgds-icon-disabled");
   }
   if (i === pageResults.length - 1) {
```

Comment:
Nit: extend this logic to disable the farRight + farLeft buttons

Reply:
already added!